omnibus/config/projects/agent.rb

```diff
@@ -18,8 +18,12 @@
   python_3_embedded "#{install_dir}/embedded3"
   maintainer 'Datadog Inc.' # Windows doesn't want our e-mail address :(
 else
+  if redhat? || suse?
+    maintainer 'Datadog, Inc <package@datadoghq.com>'
```

Comment:
Why this change (just curious)?

Reply:
When we created our new key, we changed the associated maintainer's name to this. And the signing fails if the `maintainer` declared here is not the same as the one in the key.